tests/test_bootstrap.py

```diff
@@ -92,9 +92,16 @@ def test_create_replica_old_format(self, mock_cancellable_subprocess_call):
         mock_cancellable_subprocess_call.return_value = 1
         self.assertEqual(self.b.create_replica(self.leader), 1)
 
-    def test_basebackup(self):
-        self.p.cancellable.cancel()
```

Comment:
this test case is still needed, otherwise test coverage decreases

Reply:
Added